wiki/WORKFLOW.md

```diff
@@ -35,10 +35,10 @@ We follow an approach inspired by the [Rapid Cycle Workflow](https://gitlab.com/
 
 Team alignments are organized **two times a week**. They occur:
 
-+ on **Tuesdays at 10 AM**; and
-+ on **Fridays at 8:45 AM**.
++ on **Tuesday at 10 AM**; and
++ on **Friday at 8:45 AM**.
 
-If you can't be there on time, please let us the team know about it 😄 We chose this schedule because we get to see each other for in-person classes on Wednesdays and Thursdays. If relevant, we might change the team meetings timing.
+If you can't be there on time, please let us the team know about it 😄 We chose this schedule because we get to see each other for in-person classes on Wednesday and Thursday. If relevant, we might change the team meetings timing.
```

Comment:
I believe there is a wording error here, it should either be:
- Let us know
- Let the team know

Reply:
Should be fixed in 5626a5c53c3014cc41b29f26bbe69731a7a74faf.